From 2b4ebe1d11f7f99ee83718cc5ec38db19f316891 Mon Sep 17 00:00:00 2001
From: Amrit Pal Singh <amritpal.singh@nxp.com>
Date: Wed, 8 Aug 2018 19:02:35 +0530
Subject: [PATCH 027/706] arm64: dts: ls1028a: Closure for QLINUX-8089. Added
 support for PCIE interface.

(cherry picked from commit fee4a684632b5d9cb1b4c4b45207d807cbc03f53)

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 18 ++++++------------
 1 file changed, 6 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 37eec0a2bfe6..d3898f6ee164 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -277,13 +277,10 @@
 	};
 
 	pcie@3400000 {
-		compatible = "fsl,ls-pcie", "fsl,ls2085a-pcie", "fsl,ls1088a-pcie",
-			     "snps,dw-pcie";
-		reg = <0x00 0x03400000 0x0 0x80000
-			0x00 0x03480000 0x0 0x40000   /* lut registers */
-			0x00 0x034c0000 0x0 0x40000  /* pf controls registers */
+		compatible = "fsl,ls1028a-pcie";
+		reg =  <0x00 0x03400000 0x0 0x100000
 			0x80 0x00000000 0x0 0x20000>; /* configuration space */
-		reg-names = "dbi", "lut", "ctrl", "config";
+		reg-names = "regs", "config";
 		interrupts = <0 108 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
 		#address-cells = <3>;
@@ -304,13 +301,10 @@
 	};
 
 	pcie@3500000 {
-		compatible = "fsl,ls-pcie", "fsl,ls2085a-pcie", "fsl,ls1088a-pcie",
-			     "snps,dw-pcie";
-		reg = <0x00 0x03500000 0x0 0x80000
-			0x00 0x03580000 0x0 0x40000   /* lut registers */
-			0x00 0x035c0000 0x0 0x40000  /* pf controls registers */
+		compatible = "fsl,ls1028a-pcie";
+		reg =  <0x00 0x03500000 0x0 0x100000
 			0x88 0x00000000 0x0 0x20000>; /* configuration space */
-		reg-names = "dbi", "lut", "ctrl", "config";
+		reg-names = "regs", "config";
 		interrupts = <0 113 0x4>; /* aer interrupt */
 		interrupt-names = "aer";
 		#address-cells = <3>;
-- 
2.17.1

